src/dimension.h

```diff
@@ -23,6 +23,7 @@ typedef enum DimensionType
 {
 	DIMENSION_TYPE_OPEN,
 	DIMENSION_TYPE_CLOSED,
+	DIMENSION_TYPE_STATS,
```

Comment:
Why are we treating this as dimension type. I think the consensus was that we dont want to mix up this features with dimensions.

Reply:
@svenklemm This allows us to use the existing dimension restrict info stuff to remove/use these restrictions if they are not needed for the query.